ocaml/idl/datamodel_vm.ml

```diff
@@ -1442,6 +1445,7 @@ let set_NVRAM_EFI_variables = call ~flags:[`Session]
            field ~lifecycle:[Prototyped, rel_naples, ""] ~qualifier:StaticRO ~ty:(Map(String, String)) "NVRAM"
              ~default_value:(Some (VMap []))
              "initial value for guest NVRAM (containing UEFI variables, etc). Cannot be changed while the VM is running";
+           field ~qualifier:StaticRO ~ty:Bool ~default_value:(Some (VBool false)) "suspended" "creates the VM in Suspended state";
```

Comment:
Wouldn't be better to add a `power_state` parameter instead and rejects if the value is not authorized (`Halted` or `Suspended`)?

Reply:
Agreed. VM already has a `power_state` member. It would be better to make that `StaticRO` so that it becomes a constructor argument.